js/utils.js

```diff
@@ -1311,7 +1475,10 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
       return {
         column: option.column,
         condition: option.logic,
-        value: option.value
+        filterModifier: option.filterModifier,
+        valueType: option.valueType,
+        value: option.value,
+        timezone: option.timezone
```

Comment:
Where does the `option.timezone` value come from?

Reply:
In this place this variable is unnecessary